docs/explanations/architecture.md

```diff
@@ -9,26 +8,7 @@ Blueapi performs a number of tasks:
 
 These responsibilities are kept separate in the codebase to ensure a clean, maintainable architecture.
 
-## Key Components
-
-
-![blueapi architecture main components](../images/blueapi-architecture.png)
-
-
-### The `BlueskyContext` Object
-
-Holds registries of plans and devices as well as a number of helper methods for 
-registering en-masse from a normal Python module. 
-
-### The Worker Object
-
-Wraps the Bluesky `RunEngine` and accepts requests to run plans. The requests include the name 
-of the plan and a dictionary of parameters to pass. The worker validates the parameters against
-the known expectations of the plan, passes it to the `RunEngine` and handles any errors.
-
+![blueapi main components](../images/blueapi.png)
 
-### The Service Object
 
-Handles communications and the API layer. This object holds a reference to the worker 
-can interrogate it/give it instructions in response to messages it receives from the message
-bus. It can also forward the various events generated by the worker to topics on the bus.
+Above are the main components of blueapi. The main process houses the REST API and manages the subprocess, which wraps the `RunEngine`, devices and optional connection to the downstream message broker.
```

Comment:
```suggestion
Above are the main components of blueapi. The main process houses the REST API and manages the subprocess, which wraps the `RunEngine`, devices and external connections.
```
Do we want to be general or specific?

Reply:
I made this a bit more general because I didn't think a list of major classes and their purposes was _that_ useful and was prone to being out of date. It may be more useful when blueapi is more stable.

Reply:
Lost in translation- I'm suggesting a change to make it slightly more generic but hopefully still make sense?